.snippets/text/xcm/xcm-precompile-multilocation.md

```diff
@@ -1,7 +1,7 @@
 ```js
  struct Multilocation {
     uint8 parents;
-    bytes [] interior;
+    bytes[] interior;
```

Comment:
haha 🕵️‍♀️ - but I think it works both ways

Reply:
yeah just figured I would just be consistent with changes made in the moonbeam repo 🤷‍♀️